src/components/TextEditor.vue

```diff
@@ -380,11 +410,42 @@ export default {
 				throw new Error('Impossible to execute a command before editor is ready.')
 			}
 		},
+		insertTextBlock(textBlock, addTriggrer = true) {
+			if (addTriggrer) {
+				this.appendToBodyAtCursor('!')
+			}
+			let content = textBlock.content
+			if (!this.html) {
+				const text = new Text('html', content)
+				content = toPlain(text).value
+			}
+			this.editorInstance.execute('insertItem', { content, isHtml: this.html }, '!')
+		},
 	},
 }
 </script>
 
 <style lang="scss" scoped>
+
+.bordered {
+	margin-top: var(--default-grid-baseline);
+	border: var(--border-width-input, 2px) solid var(--color-border-maxcontrast);
+    border-radius:var(--border-radius-large);
+	padding: 1px 9px;
+	&:focus {
+		border-color: var(--color-main-text);
+		border-width: var(--border-width-input-focused, 2px);
+	}
+	&:hover {
+		border-color: var(--color-main-text);
+		border-width: var(--border-width-input-focused, 2px);
+	}
+}
+.bordere:active *{
```

Comment:
typo or unused. I don't see a `bordere`

if it's `bordered` then move it into the block avobe, like you did with `:hover`